scripts/approvalTests.py

```diff
@@ -73,9 +73,9 @@
 overallResult = 0
 
 def diffFiles(fileA, fileB):
-    with open(fileA, 'r') as file:
+    with open(fileA, 'r', errors='surrogateescape') as file:
```

Comment:
`surrogateescape` does not exist in python 2, though

Reply:
I wasn't expecting it to be yet it appeared to work, but I think I must have done something wrong because I just tried again and it didn't.

I'll see what I can do...

Reply:
IIRC you have to use `io` module for python 2.7 (same as for working with encodings), so you have to make a small polyfill over `open`/`io` usage.

Reply:
Well the python 2 case doesn't actually need to worry about the encoding but a polyfill is the answer yes and I've done that now.